cmd/openshift-install/gather.go

```diff
@@ -154,9 +155,32 @@ func gatherBootstrap(bootstrap string, port int, masters []string, directory str
 		logrus.Info("Pulling VM console logs")
 		if err := consoleGather.Run(); err != nil {
 			logrus.Infof("Failed to gather VM console logs: %s", err.Error())
+		} else {
+			archives[serialLogBundlePath] = "serial"
 		}
 	}
 
+	clusterLogBundlePath, err := pullLogsFromBootstrap(gatherID, bootstrap, port, masters, directory)
+	if err != nil {
+		logrus.Infof("Failed to gather bootstrap logs: %s", err.Error())
+	} else {
+		archives[clusterLogBundlePath] = ""
+	}
+
+	if len(archives) == 0 {
+		return "", fmt.Errorf("failed to gather VM console and bootstrap logs")
+	}
+
+	logBundlePath := filepath.Join(directory, fmt.Sprintf("log-bundle-%s.tar.gz", gatherID))
+	err = serialgather.CombineArchives(logBundlePath, archives)
+	if err != nil {
+		return "", errors.Wrap(err, "failed to combine archives")
```

Comment:
```suggestion
		return "",fmt.Errorf("failed to combine archives: %w", err)
```

Reply:
Because I want to backport this, I thought about replacing the deprecated errors lib in a follow-up PR.